admin/MasterMirroring.html.md.erb

```diff
@@ -126,10 +131,14 @@ If the the log synchronization process between the master and standby has stoppe
 (1 row)
 ```
 
-To resynchronize the standby with the master, execute the following command on the master.
+To resynchronize the standby with the master:
 
-```shell
-$ hawq init standby -n
-```
+1. If you use Ambari to manage your cluster, follow the instructions in [Removing the HAWQ Standby Master](ambari-admin.html#amb-remove-standby).
```

Comment:
Shouldn't this Ambari xref point to a standby init procedure, rather than a "removing the standby" procedure?